packages/DynamicHyperlinkTransformer/src/DynamicHyperlinkTransformer.scss

```diff
@@ -10,30 +10,32 @@ a[data-dynamic-hyperlink] {
   border-radius: $button--border-radius;
   box-shadow: 0 0 2px $color--black;
   color: $color--blue-darken-10;
+  cursor: default;
```

Comment:
Since you said this is still under the clickable link, should the cursor be a pointer as before?

Reply:
That is up for debate.
I didn't like the hand cursor appearing when hovering over the error message/type on the right hand side of the card.